metals/src/main/scala/scala/meta/internal/implementation/ImplementationProvider.scala

```diff
@@ -457,6 +468,12 @@ object ImplementationProvider {
       .find(sym => sym.symbol == symbol)
   }
 
+  def isTypeAlias(
+      symbol: String,
+      findSymbol: String => Option[SymbolInformation]
```

Comment:
We could use the already existing `implDocument` and have it passed here. Then:
`ImplementationProvider.findSymbol(symbol, implDocument).exists(_.kind == SymbolInformation.Kind.TYPE)`